config/webpack/loaders/devSitePropsTable.js

```diff
@@ -125,7 +125,7 @@ const loader = async function loader(content) {
   // ensure src exists
   return this.resolve('', source, async (err, result) => {
     if (err) {
-      return callback(err);
+      return callback(new Error(`A non transpiled source file is required for the props table to be generated:\n${source}\n${err}`));
```

Comment:
What is the error if we log like:
```suggestion
      return callback(new Error(`A non transpiled source file is required for the props table to be generated:\n${source}`));
```
To me, this is quite long and difficult to determine what went wrong

Reply:
I agree, you kind of miss the first error.

Reply:
The would error look like this: 
```ERROR in ./lib/terra-dev-site/test/loaders/PropsTableExample.js?dev-site-props-table
Module build failed (from ./config/webpack/loaders/devSitePropsTable.js):
Error: A non transpiled source file is required for the props table to be generated:
/Users/dm068655/Documents/terra-dev-site/src/terra-dev-site/test/loaders/PropsTableExample
```
I agree the error being that large makes it hard to determine what is happening.

Reply:
I think the smaller error would be easier to read & still provided info as to what file needs to be fixed to fix the error